ci-operator/config/openshift/cluster-kube-descheduler-operator/openshift-cluster-kube-descheduler-operator-master.yaml

```diff
@@ -11,13 +15,13 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.14
 images:
 - from: base
   inputs:
-    openshift_release_golang-1.13:
+    openshift_release_golang-1.14:
       as:
-      - registry.svc.ci.openshift.org/openshift/release:golang-1.13
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.14
```

Comment:
Hi,
why some have this URL and most have 
registry.svc.ci.openshift.org/ocp/builder ?

i saw this discussion once
https://github.com/openshift/release/pull/10224#discussion_r454412103